### ISSUE TITLE
Links must have discernible text

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#2799]
**Record page**

impact: serious

link-name: Links must have discernible text (serious)
https://dequeuniversity.com/rules/axe/4.3/link-name?application=axeAPI

Selector: #startOverLink
HTML: `<a class=""catalog_startOverLink btn btn-primary"" id=""startOverLink"" href=""/""><span class=""icon-refresh"" aria-hidden=""true""></span> <span class=""d-none d-lg-inline"">Start over</span></a>`
Fix all of the following:
- Element is in tab order and does not have accessible text
Fix any of the following:
- Element does not have text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute

Update https://github.com/pulibrary/orangelight/blob/main/spec/features/accessibility_spec.rb#L28

**Comment by sandbergja** [comment]
Is this ticket still valid?  This link does have text ("Start over"), and it is announced by voiceover.

**Comment by kevinreiss** [comment]
Per Jane's comment we can close this one.
